src/App.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+} from '@material-ui/core';
+
+import { GoodsList } from './components/GoodsList';
+import goodsImages from './api/goodsImages.json';
+
+import './App.scss';
+
+import {
+  getAll,
+  get5First,
```

Comment:
```suggestion
  getFirstFive,
```
краще не використовувати цифри в назвах змінних